nbconvert/exporters/tests/test_asciidoc.py

```diff
@@ -39,3 +43,25 @@ def test_export(self):
         """
         (output, resources) = ASCIIDocExporter().from_filename(self._get_notebook())
         assert len(output) > 0
+        
+    @dec.onlyif_cmds_exist('pandoc')
+    def test_export(self):
+        """
+        Can a ASCIIDocExporter export something?
+        """
+        no_prompt = {
+            "TemplateExporter":{
+                "exclude_input_prompt": True,
+                "exclude_output_prompt": True,
+            }
+        }
+        c_no_prompt = Config(no_prompt)
+        exporter = ASCIIDocExporter(config=c_no_prompt)
+        (output, resources) = exporter.from_filename(
+            self._get_notebook(nb_name="prompt_numbers.ipynb"))
+            
+        in_regex = r"In&nbsp;\[(.*)\]:"
+        out_regex = r"Out\[(.*)\]:"
+
+        assert not re.findall(in_regex, output)
+        assert not re.findall(out_regex, output)
```

Comment:
Do you want to add a corresponding test that the in and out prompts are produced with the default config? It looks like we already have such tests for HTML and Latex, but not for asciidoc.

Reply:
Sure!

Reply:
done!